### PR TITLE
Add new Create wizard security and registry access workflow

### LINKED ISSUE #296: VCH wizard no-tls-verify workflow
**Details:**
The workflow needs to be re-examined as it seems a bit confusing around the secure settings. 

Both the "Enable secure access to this VCH" and "Clients will need to authenticate" do the same thing (turn off no-tls-verify).

screen shot 2018-01-05 at 2 55 55 pm

Also @pdaigle mentioned that there should be a way to disable secure access but still have access to update the "Registry Access"

screen shot 2018-01-05 at 2 58 57 pm

Need to work with UX on this to refine this workflow.
cc: @lweitzman

### PR DESCRIPTION
**Pull request by cristianfalcone**
Fixes #296

### CODE REVIEW

**Comment by cristianfalcone** [comment]
This should invalidate #161, since "Enable secure access to this VCH" is not shown anymore.

File: h5c/vic/src/vic-webapp/src/app/create-vch-wizard/registry-access/registry-access.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
+/*
+ Copyright 2017 VMware, Inc. All Rights Reserved.
```
Comment (jooskim) [inline h5c/vic/src/vic-webapp/src/app/create-vch-wizard/registry-access/registry-access.component.spec.ts]: This is minor but year should be 2018.

File: h5c/vic/src/vic-webapp/src/app/create-vch-wizard/registry-access/registry-access.component.ts [hunk 2]
```diff
@@ -0,0 +1,220 @@
+/*
+ Copyright 2017 VMware, Inc. All Rights Reserved.
```
Comment (jooskim) [inline h5c/vic/src/vic-webapp/src/app/create-vch-wizard/registry-access/registry-access.component.ts]: Ditto

**Comment by jak-atx** [comment]
@jooskim @cristianfalcone please do not merge issues that are not complete. Part of being complete is that we are conforming to the mockups and there are missing details on this one. We have to do a better job of QA'ing PRs before they get merged.

**Pull request merged**